src/test/java/org/rulez/demokracia/pdengine/VoteDeleteValidationTest.java

```diff
@@ -43,6 +43,19 @@ public void invalid_adminKey_is_rejected() throws ReportedException {
 	@tested_operation("delete vote")
 	@tested_behaviour("validate inputs")
 	@Test
+	public void a() throws ReportedException {
```

Comment:
No meaningful test name.
This contradicts to the behaviour "deletes the vote with all parameters, choices, ballots and votes cast"
https://adadocs.demokracia.rulez.org/PDEngine/edemo/master/index.html#4c75a22b-39e3-493b-b874-505eaf816282

Reply:
Fixed